bin/validate-data-package-json

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Script to validate a local JSON file against a schema specified in a GitHub repository.
+# Usage: validate-data-package-json.sh [-r ref | -l local_schema_location] [-f datapackage_file]
```

Comment:
this script does not have a `.sh` suffix, despite what the usage says

Reply:
Attempting to run this script on the sample datapackage.json results in a validation error:

```sh
./bin/validate-data-package-json -l spec -f samples/template/TIDES/datapackage.json           (issue-40-validate-example-data)TIDES
Validating data package file in 
JSON/YAML string must be a a map, cannot load "spec/tides-data-package.json"
```

Am I doing something wrong, or is there a problem somewhere else?

Reply:
It also fails without specifying `-l`

```sh
./bin/validate-data-package-json  -f samples/template/TIDES/datapackage.json                  (issue-40-validate-example-data)TIDES
Validating data package file in 
Schema file not found on GitHub for the specified TIDES version: 
```